jobs/build-charms/builder_launchpad.py

```diff
@@ -185,11 +195,15 @@ def _lp_build_download(self, build, dst_target: Path) -> Artifact:
         charm_file = self._lp_charm_filename_from_build(build)
         dl_link = build.web_link + f"/+files/{charm_file}"
         with urllib.request.urlopen(dl_link) as src:
-            target = dst_target / charm_file
-            with target.open("wb") as dst:
-                dst.write(src.read())
-            self.echo(f"Downloaded {build.title}")
-            return Artifact(Arch[build.title.split(" ")[0].upper()], target)
+            buffer = src.read()
+
+        md5sum = hashlib.md5(buffer).hexdigest()
+        target = dst_target / charm_file
+        with target.open("wb") as dst:
+            dst.write(buffer)
+
+        self.echo(f"Downloaded {build.title} md5sum={md5sum}")
```

Comment:
confirmation with md5sum we're downloading the same file which can be retrieved via launchpad webui